src/components/App.jsx

```diff
@@ -1,14 +1,183 @@
+            You've reached the QTC Mapping-DBQs API.
+            Please select a page from the navigation menu above in order to create a mapping, 
+            generate a mapping in dev mode, or run tests on a pre-mapped DBQ.
+            Please reach out with any questions or concerns.
+        </div>
+    )
+}
 
-const App = () => <div>This is {applicationPresentationName}</div>
+/*
+ * CreatePage: React frontend for --create functionality.
+ *
+ * Prompts the user for a DBQ number, calls the corresponding mapping_data_frames 
+ * API endpoint, and downloads the resulting file.
+*/
+class CreatePage extends Component {
+    constructor(props) {
+        /* Initializes app's dbqnum property and binds form event functions. */
+        super(props);
+        this.state = { dbqnum: '' };
 
-export default hot(module)(App)
+        this.handleChange = this.handleChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    handleChange(event) {
+        /* When input field changes, updates DBQ#. */
+        this.setState({ dbqnum: event.target.value });
+    }
+
+    async handleSubmit(event) {
+        /* When form is submitted, fetches mapping file from API and downloads it. */
+        event.preventDefault();
+        // catches blank input error
+        if (this.state.dbqnum == '') {
+            alert('Error: Please enter a valid DBQ number.');
+            this.setState({ dbqnum: '' });
+            return;
+        }
+        try {
+            // call to API
+            var requestURL = API_URL + 'mapping_data_frames/' + this.state.dbqnum;
+            const response = await fetch(requestURL);
+
+            // turn file into blob and create url pointing to it
+            var blobResponse = await response.blob();
+            const blobURL = window.URL.createObjectURL(new Blob([blobResponse]));
+            const blobLink = document.createElement('a');
+            blobLink.href = requestURL;
+            blobLink.setAttribute('download', this.state.dbqnum);
+            // auto-click link to download file, then cleanup url
+            document.body.appendChild(blobLink);
+            blobLink.click();
+            blobLink.parentNode.removeChild(blobLink);
+        } catch (error) {
+            alert('Error: Please enter a valid DBQ number.');
+        }
+        // reset form
+        this.setState({ dbqnum: '' });
+    }
+
+    render() {
+        /* Renders HTML form for user to enter a DBQ# and its mapping file. */
+        return (
+            <div className='app-page' id='create-page-div'>
+                <h2>Create a Mapping</h2>
+                Please enter a valid DBQ number below to generate its mapping file.
+                <form onSubmit={this.handleSubmit} autocomplete="off">
+                    <br></br>
+                    <div className="field">
+                        <label className="label">DBQ Number:</label>
+                        <div className="control">
```

Comment:
Cleanup unused class `control`. There are others as well; search and cleanup all.

Reply:
done!